ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/command/CommandProducerActionManager.java

```diff
@@ -99,7 +99,7 @@ private void start(Set<CommandProducer> commandProducers) {
 
         commandActionsPopUpGroup = new DefaultActionGroup("Commands", true, actionManager);
         actionManager.registerAction("commandActionsPopUpGroup", commandActionsPopUpGroup);
-        commandActionsPopUpGroup.getTemplatePresentation().setSVGResource(resources.execute());
```

Comment:
Please, remove the method `resources.execute()` and the related SVG resource
If you don't use `resources.execute()` and don't need it anymore.
Because that resource will be will be as rubbish after merging this PR.